frontend/src/components/Form.js

```diff
@@ -89,11 +89,14 @@ export default function Form(props) {
   const [matchPreference, setMatchPreference] = React.useState('none');
 
   /**
-   * @return {Boolean} true/false based on completion status of personal fields
+   * Method that controls the disabled attribute of MatchPreference radio group
+   * based on the completion status of the personal preference fields
+   * @return {Boolean} true/false where true means the MatchPreference options
+   * will be disabled and false means that the options will be enabled.
    */
-  function checkPersonalPreferenceFields() {
+  function toggleMatchPreferenceOptions() {
```

Comment:
A method that returns true when something should be turned off and false when it should be turned on seems counterintuitive. How about 

```
function shouldEnableMatchPReferences() {
  return role != '' || productArea != ''
}
```

Reply:
I understand this may be counterintuitive, however, the match preferences need to be disabled by default since the page will load with the role and PA empty and once either role or PA or both are filled out the preferences should be enabled again. Using this suggestion, the opposite would occur since role and product area will be empty to start off and so the method will return false which will cause the disabled attribute to be false. To make it clearer, I can change the one line method into a if-else statement and make the return values more intuitive, would that work?

Reply:
My concern is more that the return value of the method isn't intuitive from the method name. The use is here

```
          disabled={props.checkPersonalPreferenceFields}
```

so maybe the method could be called `shouldDisableMatchPreferenceField`. WDYT?

Reply:
Retained the original logic, changed the method name.